libsplinter/src/biome/user/store/memory.rs

```diff
@@ -76,6 +89,14 @@ impl UserStore for MemoryUserStore {
             })?;
 
         if inner.remove(id).is_some() {
+            #[cfg(feature = "biome-credentials")]
+            self.credentials_store
+                .remove_credentials(id)
+                .map_err(|err| UserStoreError::QueryError {
+                    context: "Cannot delete credentials store".to_string(),
```

Comment:
I think this error message is slightly off, maybe try rewording to `"Cannot delete {} from credentials store", id`

Reply:
Changed to `Cannot delete user {} from credentials store`